docs/source/introduction.rst

```diff
@@ -44,47 +44,28 @@ in a *transaction family*. While we expect users to build custom transaction
 families that reflect the unique requirements of their ledgers, we provide
 several core transaction families as models\:
 
-    * Validator Registry - Provides a methodology
+    * *Validator Registry* - Provides a methodology
```

Comment:
This introduction needs to be rewritten and all of these Sawtooth-specific references deleted

Reply:
Agree, but that can be done in a separate pull request, with a complete documentation update/rewrite